YoutubeExplode.Tests/YoutubeClientIntegrationTests.cs

```diff
@@ -87,15 +87,6 @@ public void YoutubeClient_GetVideoMediaStreamInfosAsync_Unavailable_Test(string
             Assert.ThrowsAsync<VideoUnavailableException>(() => client.GetVideoMediaStreamInfosAsync(videoId));
         }
 
-        [Test]
-        [TestCaseSource(typeof(Data), nameof(Data.GetVideoIds_RequiresPurchase))]
-        public void YoutubeClient_GetVideoMediaStreamInfosAsync_RequiresPurchase_Test(string videoId)
```

Comment:
What is happening with these now?

Reply:
They are still unplayable and detectable, but there's no preview video ID anymore. I will look into reworking exceptions into Unavailable and Unplayable. Unavailable videos are those that don't exist, are deleted or private (maybe some other cases), unplayable videos are those that need purchase, those that are controversial or banned in your country (also maybe some other cases). Only for video streams it matters that the video should be playable.